src/category_theory/triangulated/basic.lean

```diff
@@ -72,68 +66,50 @@ In other words, we have a commutative diagram:
 
 See https://stacks.math.columbia.edu/tag/0144.
 -/
-structure triangle_morphism (T₁ : triangle C) (T₂ : triangle C):=
+@[ext]
+structure triangle_morphism (T₁ : triangle C) (T₂ : triangle C) :=
 (trimor1 : T₁.obj1 ⟶ T₂.obj1)
 (trimor2 : T₁.obj2 ⟶ T₂.obj2)
 (trimor3 : T₁.obj3 ⟶ T₂.obj3)
```

Comment:
I'd be happier if we renamed these `hom1`, etc. I don't think there's any need to have `tri` in the name, and elsewhere we use `hom` in preference to `mor`.